google-cloud-tools-plugin/src/com/google/gct/idea/debugger/BreakpointUtil.java

```diff
@@ -41,7 +41,7 @@
    */
   @Nullable
   public static String getUserErrorMessage(@Nullable StatusMessage statusMessage) {
-    if (statusMessage == null || statusMessage.getIsError() != Boolean.TRUE) {
+    if (statusMessage == null || !Boolean.TRUE.equals(statusMessage.getIsError())) {
```

Comment:
Is there something going on with null here that prevents us from doing a simple || statusMessage.getIsError()?